examples/nginx/proxy.conf

```diff
@@ -1,32 +1,33 @@
-client_body_buffer_size 128k
+client_body_buffer_size 128k;
```

Comment:
I can't prove it... but I'm almost positive these were stripped by the git-precommit-hook originally.